### PR TITLE
[register] Check for path error before doing anything

### PR DESCRIPTION
**Pull request by Itxaka**
We were not checking for an error on the path before checking for other
things so that could lead to a runtime error.

This patch just makes sure that before doing anything we check that the
lstat of the path did not give an error, otherwise it skips that path
for any further checks

Also exit with a message on no registrationURL. I cant live without that any longer :D

Signed-off-by: Itxaka <igarcia@suse.com>

### CODE REVIEW

File: cmd/operator/register/root.go [hunk 1]
```diff
@@ -116,7 +118,8 @@ func run(config cfg.Config) {
 	registration := config.Elemental.Registration
 
 	if registration.URL == "" {
-		return
+		logrus.Fatal("Registration URL is empty")
+		os.Exit(1)
```
Comment (davidcassany) [inline cmd/operator/register/root.go:122]: I think the os.Exit(1) is not needed, as far as I understand the `logrus.Fatal` already exits 1. https://pkg.go.dev/github.com/sirupsen/logrus#Fatal

**Review by davidcassany — APPROVED** [review]
LGTM, just that I believe the os.Exit call is not needed.

**Review by fgiudici — APPROVED** [review]
Looks good :-)

File: cmd/operator/register/root.go (hunk 1 above)
Comment (Itxaka) [reply cmd/operator/register/root.go:122]: oh good catch!

**Pull request merged**